apps/air-discount-scheme/backend/src/app/modules/discount/discount.controller.ts

```diff
@@ -75,17 +78,23 @@ export class PrivateDiscountController {
   ) {}
 
   @Get('users/:nationalId/discounts/current')
-  @ApiExcludeEndpoint()
-  getCurrentDiscountByNationalId(
+  @UseGuards(IdsUserGuard, ScopesGuard)
+  @Scopes('@vegagerdin.is/air-discount-scheme-scope')
```

Comment:
Yeah good point. I'll re-review this just in case and see if I can simplify.
I went into this being careful not to disturb old settings since there is a three-way of users, that is
* airlines
* client
* admins

Reply:
Oh, I was thinking of a different file. You're right, thanks :+1: